incubator/azuremonitor-containers/templates/omsagent-deployment.yaml

```diff
@@ -92,6 +94,23 @@ spec:
         periodSeconds: 60
    nodeSelector:
     beta.kubernetes.io/os: linux
+   affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - labelSelector:
+              matchExpressions:
+                - key: type
+                  operator: NotIn
+                  values:
+                  - virtual-kubelet
```

Comment:
Why don't you make this configurable?

Reply:
eg.  https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L463-L473